tokio/src/fs/read_to_string.rs

```diff
@@ -7,6 +7,11 @@ use std::{io, path::Path};
 ///
 /// This is the async equivalent of [`std::fs::read_to_string`][std].
 ///
+/// This operation is blocking. However, tokio runs it in a
+/// background thread pool dedicated to blocking operations, using
+/// [`block_in_place`].
```

Comment:
I don't like saying "this operation is blocking" here. We should be very clear that using it does _not_ block the thread, even if it is implemented using `spawn_blocking`.

Also, it doesn't use `block_in_place`. It uses `spawn_blocking`.

Reply:
> Also, it doesn't use block_in_place. It uses spawn_blocking.

Sorry for that.

> We should be very clear that using it does not block the thread, even if it is implemented using spawn_blocking.

How would you rephrase it otherwise? 
I thought that saying, it's a blocking operation that tokio handles it on a separate pool, that's dedicated for these kind.